h/static/scripts/controllers/autosuggest-dropdown-controller.js

```diff
@@ -84,6 +84,10 @@ class AutosuggestDropdownController extends Controller {
     });
 
     this._setList(configOptions.list);
+
+
+    // Public API
+    this.setHeader = this._setHeader;
```

Comment:
I'd prefer to not mix internals and public api. Or rather, make it super obvious which is which